universal-login-sdk/test/core/services/TokensDetailsStore.test.ts

```diff
@@ -48,6 +48,7 @@ describe('INT: TokensDetailsStore', () => {
     const expectedToken = tokensDetailsStore.tokensDetails[0];
 
     expect(tokensDetailsStore.getTokenBy('symbol', expectedToken.symbol)).be.deep.eq(expectedToken);
+    expect(tokensDetailsStore.getTokenBy('symbol', expectedToken.symbol.toLowerCase())).be.deep.eq(expectedToken);
```

Comment:
lowercasing here defeats the point of the test. You should pass the value as mixed case and expect it to still come out as stored